max_soft_limit: 0.015
min_soft_limit: 0.0
max_velocity: 0.08
max_acceleration: 0.8
max_speed_percentage: 100.0
min_speed_percentage: 1.0
max_effort: 20.0
min_effort: 6.0
coeff_outpos_to_pulse: 2481230.769
coeff_effort_to_torque: 0.0305
